verification/curator-service/ui/src/components/Profile.tsx

```diff
@@ -83,6 +350,7 @@ export default function Profile(): JSX.Element {
                                     throw Error(`Unknown role ${role}`);
                             }
                         })}
+                        {user.googleID && <ChangePasswordFormInProfile />}
```

Comment:
Please change this condition to `!user.googleID` you actually want to show change password form to the users that aren't  registered with Google